CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Refactored
```

Comment:
```suggestion
### Removed
```
Sorry, for not making it clear before, but the sections should match the ones described by the [https://keepachangelog.com/en/1.0.0/](keepachangelog) pattern:

![image](https://user-images.githubusercontent.com/2728804/88458519-02126480-ce65-11ea-944f-0c275fd2fe39.png)